.github/workflows/build-and-test-toolchain.yml

```diff
@@ -119,6 +119,16 @@ jobs:
           echo "ccache-dir-path=`wslpath -w $CCACHE_DIR_PATH`" >> $GITHUB_OUTPUT
           echo "artifact-path=`wslpath -w $ARTIFACT_PATH`" >> $GITHUB_OUTPUT
 
+      - name: Group test failures
+        run: |
+          cd ~/work
+          source .github/scripts/config.sh
```

Comment:
I would use only one call here
~/work/.github/scripts/toolchain/group-gcc-test-failures.sh and will add all required logic to the script.